### ISSUE TITLE
Do we want to have a cryptocurrency section?

### ISSUE THREAD

**Issue by Mikaela** [privacytoolsIO/privacytools.io#1703]
I think I have recently seen multiple cryptocurrency suggestions or discussions and I think we need to formally decide whether we want to include them somewhere or reject/close them all.

* https://github.com/privacytoolsIO/privacytools.io/labels/%E2%82%BF%20cryptocurrency
* https://forum.privacytools.io/t/bitcoin-wallets-what-is-missing/1128?u=mikaela
* https://forum.privacytools.io/t/discussion-cryptocurrencies/1501?u=mikaela says that we have had a section for them, but don't anymore?
* https://forum.privacytools.io/t/discussion-cryptocurrency-virtual-currency-exchanges/1478?u=mikaela

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
I say yes, even though it is a deep topic it is really important to achieve a true decentralized exchange and given the current situation with cards replacing cash this going to get worse. We need a basic knowledge of e-coins.

**Comment by danarel** [comment]
I would personally say no. I think we'd be stepping outside the wheelhouse of what PT offers. I don't think we are in the business of recommending coins, wallets, etc, especially since there is so much volatility in their markets.

**Comment by hugoncosta** [comment]
Yes, there was a crypto section, I remember it had bitcoin, monero and zcash a couple of years ago. 

@danarel, volatility doesn't mean they're not usable, nothing has a stable price. Imo, cryptocurrency shouldn't be a standalone section, but part of a "everything Money/transfer of value" section. Unfortunately, I believe that it would end up a crypto section since there are no other open source platforms to transfer a currency of sorts (?).

I too support that. Someone really taking in everything on the website should take into account the need to pay for the services in an anonymous/pseudonymous way. How would we go about explaining everything crypto though? Plugin wallets like [Metamask](https://metamask.io/)?

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> Yes, there was a crypto section, I remember it had bitcoin, monero and zcash a couple of years ago.
> 
> @danarel, volatility doesn't mean they're not usable, nothing has a stable price. Imo, cryptocurrency shouldn't be a standalone section, but part of a "everything Money/transfer of value" section. Unfortunately, I believe that it would end up a crypto section since there are no other open source platforms to transfer a currency of sorts (?).
> 
> I too support that. Someone really taking in everything on the website should take into account the need to pay for the services in an anonymous/pseudonymous way. How would we go about explaining everything crypto though? Plugin wallets like [Metamask](https://metamask.io/)?

I agree, besides I don't see how volatility is related to privacy. We are not recommending cryptocurrencies as some sort of entrepreneur bullshit, but rather what they represent/could represent as a form of emancipation from current not-fungible currencies. 

I think that creating a "Money" category or something like that would be great, it could contain information about how credit/debit card tracking is done, and the coming cashless society.

What exactly do you mean by explaining everything crypto? Which parts?

**Comment by hugoncosta** [comment]
@5a384507-18ce-417c-bb55-d4dfcc8883fe, exactly what I had in mind, more a "this is how companies can track you via current payment methods" instead of a "here are all the alternatives" like we do on other sections.

Mostly, I was thinking of how much we want to cover related to crypto. Would we just advise on the privacy/security issues or would we dedicate more into how to use them, where to get them? For example, buying bitcoin anonymously is certainly something I would give tremendous value to as a user.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
>@5a384507-18ce-417c-bb55-d4dfcc8883fe, exactly what I had in mind, more a "this is how companies can track you via current payment methods" instead of a "here are all the alternatives" like we do on other sections.

It would also justify a bit more the section since some people have doubts about recommending cryptocurrencies.

>Mostly, I was thinking of how much we want to cover related to crypto. Would we just advise on the privacy/security issues or would we dedicate more into how to use them, where to get them? For example, buying bitcoin anonymously is certainly something I would give tremendous value to as a user.

I think it is needed to cover at least some basics stuff, since buying from an exchange with KYC could de-anonymize users for example.  Cryptocurrencies, secure FLOSS software wallets (for desktop and mobile), exchanges and explaining what KYC (know your customer) is under it are some basic stuff that can be explained simply. Then the security, anonymity and privacy measures can be mentioned under each crypto with a link redirecting to the wiki maybe were it can be further explained.

I don't think buying BTC anonymously is possible at all, but maybe I'm wrong.

**Comment by JonahAragon** [comment]
Are any of these cryptocurrencies particularly good at privacy specifically? There's like a million of them, how are we to choose a frontrunner? This is kind of one of those things everyone has a different opinion on just based on their own biases.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> Are any of these cryptocurrencies particularly good at privacy specifically? There's like a million of them, how are we to choose a frontrunner? 

I don't know if there should be a frontrunner, maybe we could only mention the privacy features of specific coins. I think Monero, Zcash, Komodo and Pirate Chain are the only coins with certain aspects that provide privacy.

>This is kind of one of those things everyone has a different opinion on just based on their own biases.

I couldn't disagree more on this.  I know the world of crypto is full of capitalist propaganda towards creating a business out of holding and glorifying whichever of the twenty thousands coins, tokens or whatever you have chosen to create a cult around, but there are actual ways to measure how private or anonymous one can be. If you read my post about ARRR you will see mention about different technologies that they apply to create a private/anonymous environment, which have been adopted from other coins such as Monero, Zcash, Komodo, etc.  I am not saying Pirate Chain is the best solution ever, I am trying to show it as a comparison so you can see how this is achieved, I am not a security or cryptographer expert so maybe I'm missing a few things but so far the only flaw I have seen is that a 51% attack on BTC is not ***so*** impossible and it can affect it.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
>There's like a million of them, how are we to choose a frontrunner?

Also, I don't know if this is the best approach for a technology in development such as this since the improvements over time are slow and require trial and error with real life scenarios since it is really hard to predict what will happen.  But I think that creating some guidelines as to what it is a "privacy coin" could help to go in the right way, since right now Dash is considered a privacy coin and I can't see anything on it that resembles that.

The most important question IMHO is,  do we think they have a real potential at emancipation in society or are they going to fail and we will be forced either create a new software or get used to credit/debit card payments and ad tracking?

**Comment by lrq3000** [comment]
If I can give my humble opinion, I think a "money" or "funds transfer" section is important. Without financial independence, there is no true privacy nor anonymity. That's a very critical point of failure for any privacy-oriented system, and hence as we can observe an area of control that interests all governments (historically, this is one of the first things a government does, to control the money production and transfers).

> The most important question IMHO is, do we think they have a real potential at emancipation in society or are they going to fail and we will be forced either create a new software or get used to credit/debit card payments and ad tracking?

Some of them already works, the technology exists and it's enough to prove it's possible and to ensure such systems will exist (maybe in another form) in the future, whether or not the current ones end up fading away.

Also, a good privacy-oriented cryptocurrency (or fund transfer system, let's not limit to only cryptocurrencies, even if I have a hard time imagining any other system to do that) provides architectural guarantees of privacy, just like Tor or other self-contained networks can do. One of them is I think Monero, there may also be others, but I did not study them enough to say anything (eg, I think Loki intends to provide anonymous transactions too, just like for messages in their messaging client).

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
>If I can give my humble opinion, I think a "money" or "funds transfer" section is important. Without financial independence, there is no true privacy nor anonymity. That's a very critical point of failure for any privacy-oriented system, and hence as we can observe an area of control that interests all governments (historically, this is one of the first things a government does, to control the money production and transfers).

Couldn't agree more, fiat is centralized, and on top of cards is the perfect behavioral tracking mechanism, sooner or later is going to become the only way if we do not change it.

>Some of them already works, the technology exists and it's enough to prove it's possible and to ensure such systems will exist (maybe in another form) in the future, whether or not the current ones end up fading away.

Yes, I know they work, although they might have some flaws.  It was a rhetoric question to Jonah, since he is a bit skeptic about their value as a private alternative.

>Also, a good privacy-oriented cryptocurrency (or fund transfer system, let's not limit to only cryptocurrencies, even if I have a hard time imagining any other system to do that) provides architectural guarantees of privacy, just like Tor or other self-contained networks can do. One of them is I think Monero, there may also be others, but I did not study them enough to say anything (eg, I think Loki intends to provide anonymous transactions too, just like for messages in their messaging client).

Yes, I also think Monero is a good one, not sure how much anonymity the ring signatures provide, and I think they have optional transparent transactions, which could affect fungibility, but I haven't looked enough.  Is loki planning to build a new crypto or how do they want to do that?

**Comment by iteco** [comment]
> If I can give my humble opinion, I think a "money" or "funds transfer" section is important. Without financial independence, there is no true privacy nor anonymity. That's a very critical point of failure for any privacy-oriented system, and hence as we can observe an area of control that interests all governments (historically, this is one of the first things a government does, to control the money production and transfers).

Fully agreed too.

**Comment by blacklight447-ptio** [comment]
One thing we need to consider, the world of crypto currency is very volatile and there is a  LOT of FUD about it out there. 

While on one hand, us providing trustworthy and stable information about it would be great. On the other hand, I am not aware of anyone on our team with the proper experience to write it and maintain it. 

So we would have to attract someone to the team with the needed expertise, which we can also trust to give an unbiased, well thought out advise.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> One thing we need to consider, the world of crypto currency is very volatile and there is a LOT of FUD about it out there.
> 
> While on one hand, us providing trustworthy and stable information about it would be great. On the other hand, I am not aware of anyone on our team with the proper experience to write it and maintain it.
> 
> So we would have to attract someone to the team with the needed expertise, which we can also trust to give an unbiased, well thought out advise.

I think that's fine, I understand your point, I know they are volatile but, again, I think a warning should be enough for that, the idea is not to recommend people where to invest but which cryptos can be used instead of cards. I also agree on the FUD, it is really hard to find good information about this topic since most of them is people talking about their preferences or the coin they want to make propaganda, that's why I think it would be really important if someone could provide accurate information about this technology.

I am making a comparison table on the wiki, you all can take a look at it to see how some currencies can provide a better privacy than others. It is still under construction, though.

https://wiki.privacytools.io/view/Comparison_of_cryptocurrencies

**Comment by iteco** [comment]
Perhaps it could be also crowd sourced here or some other forum. There are - at the moment - only one or few notable anonymous crypto currencies. I guess that people here could easily come to an agreement about the list. 

Monero and Zcash have a [recommendation](https://cryptobriefing.com/snowdens-crypto-privacy/) from Snowden. They have marketcap (Monero) or volume (Zcash) in top 20 according [Coinmarketcap](https://coinmarketcap.com/), and both projects are rated to have A-class health ([Monero](https://coinmarketcap.com/currencies/monero/ratings/) | [Zcash](https://coinmarketcap.com/currencies/zcash/ratings/)). 

There are lot of [other projects](https://coinsutra.com/anonymous-cryptocurrencies/) too, but they are not so stable yet.

**Comment by lrq3000** [comment]
@5a384507-18ce-417c-bb55-d4dfcc8883fe Great table, thanks a lot! So from your preliminary overview, it looks like Monero and Pirate Chain offer the most privacy and anonymity by default? It's weird that Zcash and Zcoin which claim to be anonymous do not provide such transactions by default.

Also maybe Loki could be added? (or for the moment clustered with Monero, since it's mostly a fork for now?).

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
> @5a384507-18ce-417c-bb55-d4dfcc8883fe Great table, thanks a lot! So from your preliminary overview, it looks like Monero and Pirate Chain offer the most privacy and anonymity by default? It's weird that Zcash and Zcoin which claim to be anonymous do not provide such transactions by default.
> 
> Also maybe Loki could be added? (or for the moment clustered with Monero, since it's mostly a fork for now?).

Yes, Monero, Pirate Chain and also Komodo (which is a fork of Zcash which enables private transactions by default; also, ARRR is a fork of Komodo FYI).

I haven't thought about Loki, I haven't had time to look at their crypto, I just focused on their messenger, but will try to take a look today and see if I can add it, but I want to first finish with Komodo since it is quite relevant and Loki is just beginning. Still, I think they deserve its own space, I'm sure there must be some differences between XMR and Loki.

**Comment by lrq3000** [comment]
Yes Loki is just starting indeed, but they already have a functional application, the Session messenger, so I think it's already further than most other coins. Plus, although there are tons of altcoins, there aren't so many "anonymous by default" coins.

**Comment by BTCparadigm** [comment]
Thoughts from a Bitcoiner:

- CACK for cryptocurrency/money/payment section
- Many coins can have good privacy tech but it's useless if the coins doesn't have liquidity or acceptability, which makes many of the previously mentioned coins close to useless.
- Money is the most liquid good, that also stores value thought space and time. That's Bitcoin. 
- Payment method/medium of exchange can be whatever the parties accept to use. AFAIK for example Tails and DNMs have mostly ended up supporting BTC and XMR. Why? Because they actually work.
- You can use Bitcoin in a private way with the correct software (Wasabi Wallet & JoinMarket) but unfortunately most wallets use bad privacy practices.
- Monero is relatively private by default but it comes with a security and verifiability trade off. Might still be concidered good enough for payments. 
- Snowden really doesn't know what he was talking about.
- Zcash has good privacy but it's optional, which means less than 5% use that feature AFAIK.

**Comment by LongJohn-Silver** [comment]
BTC will become a mess now that the Chinese government is beginning to shut down all mining operations to acomplish their zero emissions goal.

**Comment by lrq3000** [comment]
I have another perspective. IMHO, [the delisting of Monero from big exchanges earlier this year and the bounty set by US government to break Monero's anonymity](https://cointelegraph.com/news/regulators-dial-up-the-heat-dash-zec-and-monero-reach-boiling-point) are proofs that Monero is indeed private enough to warrant concerns from surveillance agencies. Also now Monero has a new "network’s new zero-knowledge proving system" called [Bulletproofs+ which was successfully audited](https://cointelegraph.com/news/atomic-swaps-and-network-upgrades-send-monero-xmr-price-to-a-3-year-high). There are also several decentralized exchanges that allow to buy/sell it (eg, [new Monero dex Haveno, BTC/XMR atomic swaps](https://cointelegraph.com/news/atomic-swaps-and-network-upgrades-send-monero-xmr-price-to-a-3-year-high), cross-chain dexes such as BlockDX, ....), so I don't think this is particularly an issue.

About BTC, although I do not think it's going to be much phased by what the Chinese government is trying to do, for now anonymous transactions only work through specific apps, although there are plans to make the lightning network layer more anonymous IIRC.

Another new contender in this field, launched just a few months ago this year, is the [Secret Network](https://scrt.network/), which offers to anonymize transactions on all Ethereum tokens.

**Comment by lrq3000** [comment]
Anyway if PTIO is to add a cryptocurrency section, I think we need to define a clear set of prerequisites to be listed. Being open-source is not sufficient, as well as anonymity. There are other parameters such as full pre-mining, initial tokens distribution, decentralization (who runs the nodes?), etc.

**Comment by lrq3000** [comment]
I would like to suggest to continue this discussion in light of new informations that came to my knowledge.

First, traditional transaction platforms relying on central banks are privacy averse, one recent example being [PayPal banning the account of a non-profit organization that used the funds for the hosting fees of Tor nodes](https://darknetlive.com/post/paypal-closed-someone-s-account-for-running-tor-relays/).

Another info is that [Bitcoin and cryptocurrencies can be historically linked back to the CypherPunk movement](https://www.sciencespo.fr/evenements/?event=the-blockchain-ideology-after-the-crypto-boom-is-the-cypherpunk-movement-still-alive), which clearly PTIO is also inheriting from.

I think PTIO should include a cryptocurrency section. Not necessarily recommend any specific coin since there is still some debate about which has adequate and robust privacy guarantees (and the field is still evolving on that aspect), but at least a section explaining the purpose and value in terms of privacy and decentralized censorship-resistant ownership of funds. Basically the main aspects the CypherPunk movement aspired to, we could make a section around these items for a start.

**Comment by lrq3000** [comment]
PS: to extend on my idea of a section without recommending any coin, it would be an "education section" like the sections on the Five Eyes etc, to explain the utility of the blockchain technology but not necessarily what to use. Of course if at some point we agree what coin can be recommended, that would be great, but at this current point in time, there is at least something that is certain, that there is no doubt about the viability of the blockchain technology and how it fulfills several of the aims of the CypherPunk movement.
